proto/MODULE.bazel

```diff
@@ -0,0 +1,8 @@
+module(
+    name = "rules_python_proto",
+    compatibility_level = 1,
+    version = "0.0.0",
+)
+
+bazel_dep(name = "rules_proto", version = "5.3.0-21.5")
+bazel_dep(name = "rules_python", version = "0.4.0")
```

Comment:
I thought you want to have a separate module to avoid having extra dependencies of rules_python, but here you depend on rules_python already, then having a sub module doesn't seem necessary anymore?

Reply:
The other way around. I don't want `rules_python` to depend on `rules_proto` or on `com_google_protobuf`.

Reply:
I see, that makes sense.